draft-ietf-mls-architecture.md

```diff
@@ -635,20 +625,17 @@ states, breaking their ability to communicate.
 
 ### Recovery After State Loss
 
-Group members whose local MLS state is lost or corrupted
-can reinitialize their state and continue participating in the
-group. This does not provide the member with access to group
+Group members whose local MLS state is lost or corrupted can reinitialize their
+state by re-joining the group as a new member and removing the member
+representing their earlier state.  An application can require that a client
+performing such a reinitialization proof its prior membership with a PSK.
```

Comment:
Maybe we should note here that such a mechanism can allow an adversary to circumvent PCS. For example, a client gets compromised and recovers via an update. Then the adversary decides that they still want access to the group and does a re-join. One way to mitigate this is for clients to require that the PSK presented by a re-joining client not be older than the epoch resulting from the re-joining client's last update.

Reply:
@kkohbrok I think this should go in the Protocol document, could you add a PR please?